pkg/sbproxy/reconcile/resyncer.go

```diff
@@ -14,20 +14,22 @@ import (
 
 // NewResyncer returns a resyncer that reconciles the state of the proxy brokers and visibilities
 // in the platform to match the desired state provided by the Service Manager.
-func NewResyncer(settings *Settings, platformClient platform.Client, smClient sm.Client, proxyPath string) Resyncer {
+func NewResyncer(settings *Settings, platformClient platform.Client, smClient sm.Client, smPath, proxyPathPattern string) Resyncer {
 	return &resyncJob{
-		options:        settings,
-		platformClient: platformClient,
-		smClient:       smClient,
-		proxyPath:      proxyPath,
+		options:          settings,
+		platformClient:   platformClient,
+		smClient:         smClient,
+		smPath:           smPath,
+		proxyPathPattern: proxyPathPattern,
 	}
 }
 
 type resyncJob struct {
-	options        *Settings
-	platformClient platform.Client
-	smClient       sm.Client
-	proxyPath      string
+	options          *Settings
```

Comment:
from these settings(reg/app settings) you could probably remove the username and password as they will be the same as platform user/password - we could try to avoid some confusion  that way